packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx

```diff
@@ -952,6 +964,49 @@ export class FieldEditor {
     );
   }
 
+  private renderStatusToggle(objFormValue) {
+    const strBaseClassName = 'fw-field-editor';
+    const choices = objFormValue.choices;
+    return (
+      <div class={`${strBaseClassName}-status-toggle`}>
+        <div class={`${strBaseClassName}-status-toggle-item header`}>
+          <span>{i18nText('fieldLabel')}</span>
+          <span>{i18nText('ertText')}</span>
+        </div>
+        {choices.map((dataItem) => {
+          let toggle = null;
+          if (
+            dataItem?.choice_options &&
+            Object.keys(dataItem?.choice_options).length
+          ) {
+            toggle = (
+              <fw-toggle
+                id={dataItem.id}
+                size='medium'
+                checked={dataItem?.choice_options.resolution_timer}
+                onFwChange={this.statusToggleHandler}
+              ></fw-toggle>
+            );
+          }
+          return (
+            <div class={`${strBaseClassName}-status-toggle-item`}>
+              <span>
+                <div class={`${strBaseClassName}-input-container`}>
+                  <fw-input
+                    class={`${strBaseClassName}-content-required-input`}
+                    value={dataItem.value}
+                    disabled='true'
+                  ></fw-input>
+                </div>
+              </span>
+              <span>{toggle}</span>
```

Comment:
check if `toggle` is truthy and then render the `<span>`

Reply:
moved the span to the condition above